src/features/hotspots/setup/HotspotSetupWifiConnectingScreen.tsx

```diff
@@ -46,7 +69,7 @@ const HotspotSetupWifiConnectingScreen = () => {
         showOKAlert({ titleKey: 'Your password is invalid' })
```

Comment:
I think this key should be `{ titleKey: 'generic.invalid_password' }`

Reply:
Updated